src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -27,12 +28,15 @@ var (
 func Configure(cfg *config.Config, version string) {
 
 	if cfg.AutoReport != nil && *cfg.AutoReport {
-		analytics.Configure(TrackingId, version, cfg.Addr)
+		pubsub.Sub(pubsub.IP, func(ip string) {
+			log.Debugf("Got IP %v -- starting analytics", ip)
+			analytics.Configure(ip, TrackingId, version, cfg.Addr)
```

Comment:
If the `pubsub.IP` topic takes long to be published, it could happen. But I see an interface in analytics that only allows for starting/stopping the service, so how could the UI actively call the service?